kubernetes/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM centos:6.9
```

Comment:
It is not recommended to use the operating system as the base image, you can use the base image provided by maven. Https://hub.docker.com/_/maven